tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "include": ["./packages/*/src/**/*.ts"],
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "lib": [
+      "esnext"
+    ],
+    "noEmit": true,
```

Comment:
would you like to add declarations and declaration maps later? or do you not want to publish types with these packages for now?

I noticed [Babel core had both](https://github.com/babel/babel/blob/e703e9f838459068bdc4e6b4ef119649862bbbba/tsconfig.base.json#L9), and I generally recommend using both. Not sure if your intent with `noEmit` was just to simplify things for now or something else

Reply:
I prefer not to publish the type definitions yet; we'll do it when releasing Babel 8 for the main packages.